renormalizer/mps/mps.py

```diff
@@ -181,11 +181,15 @@ def random(cls, mol_list: MolList, nexciton, m_max, percent=1.0) -> "Mps":
 
     @classmethod
     def gs(cls, mol_list: MolList, max_entangled: bool):
-        """
-        T = \\infty maximum entangled GS state
-        electronic site: pbond 0 element 1.0
-                         pbond 1 element 0.0
-        phonon site: digonal element sqrt(pbond) for normalization
+        r"""
+        Obtain ground state at :math:`T = 0` or :math:`T = \infty` (maximum entangled).
+        Electronic DOFs are always at ground state. and vibronic DOFs depend on ``max_entangled``.
```

Comment:
vibrational not vibronic

Reply:
@liwt31  You can fix it in the next commit.

Reply:
OK I'll keep this in mind